### ISSUE TITLE
TypeError: t is undefined

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/action#3196]
https://sentry.io/organizations/parabol/issues/1240359336/?referrer=github_plugin

```
TypeError: t is undefined
  at props (../client/components/EditorInputWrapper.tsx:65:11)
  at 1290/t.exports (/home/circleci/action/node_modules/draft-js/lib/editOnKeyDown.js:99:1)
  at a (/home/circleci/action/node_modules/draft-js/lib/DraftEditor.react.js:314:1)
  at b (/home/circleci/action/node_modules/react-dom/cjs/react-dom.production.min.js:243:230)
  at unstable_runWithPriority (/home/circleci/action/node_modules/scheduler/cjs/scheduler.production.min.js:18:156)
...
(19 additional frame(s) were not displayed)
```

AC
- [ ] no error on sentry

effort: 10.5

**Comment by mattkrick** [comment]
Sentry issue: [ACTION-PRODUCTION-231](https://sentry.io/organizations/parabol/issues/1233155090/?referrer=github_plugin)

**Comment by mattkrick** [comment]
at least the first instance is fixed in #3194
